qf_lib/containers/futures/future_tickers/future_ticker.py

```diff
@@ -87,16 +87,17 @@ def __init__(self, name: str, family_id: str, N: int, days_before_exp_date: int,
         self._last_cached_datetime = None
         self._expiration_hour = RelativeDelta(hour=0, minute=0, second=0, microsecond=0)
 
-    def initialize_data_provider(self, timer: Timer, data_provider: "DataProvider"):
+    def initialize_data_provider(self, timer: Timer, data_provider: "FuturesDataProvider"):
```

Comment:
For this part, the futures ticker has a timer and a data_provider, but shouldn't now the data_provider have the timer and the ticker use that timer?